front/app/containers/Admin/projects/project/inputForm/index.tsx

```diff
@@ -58,13 +58,23 @@ export const IdeaForm = () => {
   );
 };
 
+const isIdeationContext = (participationContext: string | undefined) => {
+  if (
+    participationContext === 'ideation' ||
+    participationContext === 'budgeting'
+  ) {
+    return true;
+  }
+  return false;
+};
+
```

Comment:
think you could probably make the syntax simpler here:

```suggestion
    const isIdeationContext = (participationContext: string | undefined) => {
      return participationContext === 'ideation' || participationContext === 'budgeting';
    };```

Reply:
Ah yep :sweat_smile: Went a bit fast and missed that. Thanks!!